src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { SortTypes } from '../../types/SortTypes';
+import { Nav } from '../Nav';
+
+interface Props {
+  todos: Todo[];
+  onChangeFilter: (filter: SortTypes) => void;
+  activeFilter: SortTypes;
+  onClearCompletedTodos: () => void;
+}
+
+export const Footer: FC<Props> = ({
+  todos,
+  onChangeFilter,
+  activeFilter,
+  onClearCompletedTodos,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
```

Comment:
It's better to use useMemo to avoid filtering todos on every render.

Reply:
Done